Reaktoro/Core/Phases.py

```diff
@@ -75,3 +75,157 @@ def testPhases():
     assert phases[7].name() == "Dolomite"
     assert phases[8].name() == "Quartz"
 
+    # -----------------------------------------------------------------------------------------------------------------
+    # Testing AqueousPhase with provided speciates and tags, so that species possessing them are excluded
+    # -----------------------------------------------------------------------------------------------------------------
+
+    db = Database("supcrt98.yaml")
```

Comment:
Having many calls like this will slow down the execution of the tests (i.e., `Database("longfile")`). Use the `db` object created above. Change some lines to something like `db.addSpecies ( Species(...).withTags(...) )` so that we can filter out species in the tests below.

Reply:
This can be very simple, since a more extensive test is done in the C++ interface.

Reply:
Corrected.